components/PyTorch/pytorch-kfp-components/pytorch_kfp_components/components/base/base_component.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env/python3
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+"""Pipeline Base component class."""
+import abc
+from pytorch_kfp_components.types import standard_component_specs
+from six import with_metaclass
+
+
+class BaseComponent(with_metaclass(abc.ABCMeta, object)):  # pylint: disable=R0903
```

Comment:
Unless you intend to support Python 2.7, which was sunset already more, this can be simply `metaclass=abc.ABCMeta`

Reply:
Thanks @chensun . I will change it and re-upload

Reply:
@shrinath-suresh do you want to address this comment?